civihr_default_theme/gulpfile.js

```diff
@@ -19,7 +20,7 @@ var scssLint = require('gulp-scss-lint');
 var jshint = require('gulp-jshint');
 
 // CSS.
-gulp.task('css', function() {
+gulp.task('css', ['css:sync'], function() {
```

Comment:
This is just a question: why is sync required? how is it used? (I suspect it has something to do with adding files from Shoreditch)